integration_tests/configs/byzantine.yaml

```diff
@@ -0,0 +1,26 @@
+chain_id_1:
+  validators:
+    - coins: 8000cro
+      staked: 8000cro
+      base_port: 26650
```

Comment:
I guess this is the issue, don't set port explicitly, it'll be generated automatically.

Reply:
@linfeng-crypto